doc/adoc/user_guide.adoc

```diff
@@ -254,6 +254,13 @@ If enabled, it will run the zypper migration plugin with increased verbosity.
 [listing]
 verbose_migration: true|false
 
+Enable the fix option for pre_checks::
+If enabled (default), the run_pre_checks systemd process will use the `--fix`
+option to automatically remediate applicable issues before the migration is started. 
+
+[listing]
+pre_checks_fix: true|false
```

Comment:
What happen if this is set to false? Will the service remove the -f option from the command line?

```ExecStart=/usr/bin/suse-migration-pre-checks -f -t```

Reply:
@guangyee oh yeah i need to fix that ;-) I originally had a second pre-checks script that was used by the service but it seems redundant to have two python files doing the same thing. When I removed the second script, I forgot to add the logic back in to handle this variable. Thanks for catching. I will fix this and request a re-review once done